apps/condo/domains/ticket/utils/testSchema/index.js

```diff
@@ -92,10 +94,40 @@ async function updateTestTicketStatus (client, id, extraAttrs = {}) {
     return [obj, attrs]
 }
 
+async function createTestTicketChange (client, ticket, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    if (!ticket || !ticket.id) throw new Error('no ticket.id')
+
+    const attrs = {
+        dv: 1,
+        ticket: { connect: { id: ticket.id } },
+        ...extraAttrs,
+    }
+    const obj = await TicketChange.create(client, attrs)
+    return [obj, attrs]
+}
+
+async function updateTestTicketChange (client, id, extraAttrs = {}) {
```

Comment:
Useless you don’t have any access

Reply:
They are tests for this, expecting update operation to cause error.

Reply:
ok